### DISCUSSION TITLE
how to interpret the IVIVE  for the midazolam model??

### DISCUSSION THREAD

**Discussion by AyaSaleh90** [Open-Systems-Pharmacology/Forum#771] [Questions & Problems] [answered]
Hello,
Now I'm replicating the midazolam PBPK model using IV bolus data from healthy individuals, and I have couple of questions regarding the model:

Regarding the IVIVE interpretation of the values implemented in the PK-Sim, like the metabolism by CYP3A4:

1. In vitro Vmax = 850 pmol/min/mg microsomal protein
2. Content of CYP protein in liver microsomes = 108 pmol/mg microsomal protein --> E
3. Km = 4 umol/L --> obtained from In vitro  literature
4. Kcat= 8.761 1/min --> obtained by parameter identification using PK clinical data

1st Step:

1. Vmax = Kcat*E = 946.2 pmol/min/mg microsomal protein

2. CLint  = Vmax/Km = 946.2 *10 ^-6 (umol/min/mg) / 4 umol/L = 0.000237 L/min/mg microsomal protein

3. CLint = CLint * MPPGL *average Liver weight = 0.000237 * 40 * 1500 = **14.2 L/min**

where:
MPPGL is the amount of microsomal protein per gram of liver = 40 mg microsomal protein/gm liver --> implemented in PK-Sim
Assumed  average Liver weight= 1500 gm

2nd step, if I assumed a well-stirred model for liver (assumed):
    fraction unbound in plasma= 0.031
    Assuming hepatic blood flow rate = **1.4 L/min** 

4. Hepatic clearance = Q* fu * CLint / Q + (fu * CLint) = 1.4 * 0.031 * 14.2 / 1.4 + (0.031 * 14.2) = 0.335 * 60 = **20.12 L/h**r

So this value is hepatic clearance of midazolam due to the metabolism by CYP3A4, right or I'm missing something??

Thanks in advance,
Aya

**Comment by JanSchlender** [comment, marked as answer]
Hi Ayatallah,

Apologies for the late reply. You may have a look at previous entries on this workflow:
Duplicate of #101 
Duplicate of #177
